pod/perldelta.pod

```diff
@@ -171,6 +171,38 @@ On platforms that don't support Inf/NaN values in floating-point numbers (such
 as VAX), C<builtin::inf> and C<builtin::nan> now throw a runtime error (rather
 than breaking the perl build). [GH #22882]
 
+=item *
+
+L<ExtUtils::ParseXS> has been extensively refactored internally and
+extensive tests added. Most of these changes shouldn't be visible
```

Comment:
```suggestion
extensive tests have been added. Most of these changes shouldn't be visible
```
Can't implicitly reuse previous verb "has been" because "tests" is plural (subject-verb disagreement).